CONTRIBUTING.md

```diff
@@ -141,6 +141,33 @@ to email the patch to <gap@gap-system.org>.  You can refer to
 [this tutorial](https://burzalodowa.wordpress.com/2013/10/05/how-to-send-patches-with-git-send-email/)
 on how to do this.
 
+## Reviewing code contributions
+
+When reviewing code, you can use the following list as a checklist.
+
+### Before you even run the code (see [here](https://lornajane.net/posts/2015/code-reviews-before-you-even-run-the-code))
```

Comment:
This seems to assume a code reviewer will necessarily run the code. But I actually almost never do.

BTW, reading this text, it could very well also be addressed at somebody who wants to contribute a PR, rather than review it... Which makes sense, I guess. Might be worthwhile to point this out somewhere. Or perhaps move (some of) this list into a "How to contribute code" section, and then here just say that a reviewer should check if the criteria listed in that section are met.

Reply:
I'ld add this to the end of the `Making changes` section:
> Feel free to read the section [the reviewing process](https://github.com/gap-system/gap/blob/master/CONTRIBUTING.md#the-reviewing-process) to make the review of your pull request go smoothly.
What do you think?